app/frontend/src/pages/LfgPage/LfgCard.js

```diff
@@ -1,16 +1,100 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import ReportIcon from "@mui/icons-material/Report";
 import {Button} from "@mui/material";
 import {useNavigate} from 'react-router-dom'
 import {deleteLfg} from "../../services/lfgService";
 import ClearIcon from '@mui/icons-material/Clear';
 import EditLfg from './EditLfg'
+import axios from 'axios'
 
 const LfgCard = ({ group, currentUser }) =>  {
 
     const isCurrentUserCreator = currentUser && group.user.username === currentUser.username ? true : false
+    const axiosInstance = axios.create({
+        baseURL: `${process.env.REACT_APP_API_URL}`
+    })
+
+    const isUserGroupMember = (user, group) => {
```

Comment:
You can use async here since sometimes user returns null.

Reply:
I added null check and async function.

Reply:
It seems okay now.